doc/modules/classes.rst

```diff
@@ -1200,6 +1200,7 @@ See the :ref:`metrics` section of the user guide for further details.
    preprocessing.PolynomialFeatures
    preprocessing.RobustScaler
    preprocessing.StandardScaler
+   preprocessing.TransformedTargetRegressor
```

Comment:
How about renamind to "TransformTargetRegressor" to gain 2 letters on the length of the name.